pkg/ingester/activeseries/active_labels_test.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/prometheus/prometheus/storage"
 	"github.com/prometheus/prometheus/tsdb/index"
 	"github.com/stretchr/testify/require"
+
+	asmodel "github.com/grafana/mimir/pkg/ingester/activeseries/model"
```

Comment:
If you name this package as `activeseries/asmodel` then you don't need to add an import alias in each one of the 11 files importing it.